eclipse-scout-core/src/form/fields/LookupBox.less

```diff
@@ -43,6 +43,23 @@
         top: 0;
         height: 100%;
       }
+
+      .active-filter-undefined&.inactive {
```

Comment:
This does not work with Scout Classic, AbstractListBox already adds inactive text and style